docs/proposals/pipeline/pipeline.md

```diff
@@ -368,35 +368,55 @@ For more details, see [Tekton signed-provenance-tutorial](https://tekton.dev/doc
 
 #### Kurator Pipeline cli
 
-##### 1. Pipeline Commands
+We are focused on providing the viewing logs of Pipeline executions. Therefore, we have concentrated the CLI's features on this core requirement.
 
-- **`pipeline list`**
-  
-    Display all current Pipelines.
+For viewing information about Pipeline objects, users can directly utilize existing tools like kubectl.
 
-- **`pipeline describe <name>`**
+##### Pipeline Execution Commands
 
-    Show detailed information of a specified Pipeline.
+###### List Commands
 
-- **`pipeline log --lastExecution <name>`**
+- **`kurator pipeline execution list -n`**
```

Comment:
add namespace

Reply:
I am not sure whether `get execution` or this is better to understand

Reply:
yes, namespace is needed

Reply:
> I am not sure whether `get execution` or this is better to understand

you are right, I don't like the name `run`, so I change it to `execution` . but I am not sure if it is better